apps/admin-portal/src/configs/app.ts

```diff
@@ -71,5 +76,6 @@ export const ServiceResourcesEndpoint: ServiceResourcesType = {
     user: `${GlobalConfig.serverHost}/api/identity/user/v1.0/me`,
     userStores: `${GlobalConfig.serverHost}/api/server/v1/userstores`,
     users: `${GlobalConfig.serverHost}/scim2/Users`,
-    wellKnown: `${GlobalConfig.serverHost}/oauth2/oidcdiscovery/.well-known/openid-configuration`
+    wellKnown: `${GlobalConfig.serverHost}/oauth2/oidcdiscovery/.well-known/openid-configuration`,
+    selfSignUp: `${GlobalConfig.serverHost}/api/server/v1/identity-governance/${IDENTITY_GOVERNANCE_ACCOUNT_MANAGEMENT_POLICIES_ID}/connectors/${SELF_SIGN_UP_CONNECTOR_ID}`
```

Comment:
I think this line exceeds 120 characters.

Reply:
Tried breaking this to two but then it gives an error. How to break this?

Reply:
You can use `\` to ensure line continuation. 
```suggestion
    selfSignUp: `${GlobalConfig.serverHost}/api/server/v1/identity-governance/\
    ${IDENTITY_GOVERNANCE_ACCOUNT_MANAGEMENT_POLICIES_ID}/connectors/${SELF_SIGN_UP_CONNECTOR_ID}`
```

Reply:
Or just break like bellow.

```suggestion
    selfSignUp: `${GlobalConfig.serverHost}/api/server/v1/identity-governance/${
    IDENTITY_GOVERNANCE_ACCOUNT_MANAGEMENT_POLICIES_ID}/connectors/${SELF_SIGN_UP_CONNECTOR_ID}`
```

Reply:
Fixed.